test_sum_terms.py

```diff
@@ -0,0 +1,5 @@
+Test.assert_equals(series_sum(1), "1.00")
```

Comment:
Have you tried actually running these tests on your machine? You don't actually have a `Test` class imported anywhere.